jobs/promptLicenceCreation.ts

```diff
@@ -0,0 +1,152 @@
+import 'reflect-metadata'
+import moment from 'moment'
+import _ from 'lodash'
+import { buildAppInsightsClient, flush, initialiseAppInsights } from '../server/utils/azureAppInsights'
+import logger from '../logger'
+import { Prisoner } from '../server/@types/prisonerSearchApiClientTypes'
+import config from '../server/config'
+import { services } from '../server/services'
+import { ManagedCase } from '../server/@types/managedCase'
+import LicenceStatus from '../server/enumeration/licenceStatus'
+import { EmailContact } from '../server/@types/licenceApiClientTypes'
+import { convertToTitleCase } from '../server/utils/utils'
+
+initialiseAppInsights()
+buildAppInsightsClient('create-and-vary-a-licence-prompt-licence-create-job')
+
+const { caseloadService, prisonerService, communityService, licenceService } = services
+
+const pollPrisonersDueForLicence = async (): Promise<Prisoner[]> => {
+  const getEligiblePrisoners = async (prisonCode: string): Promise<Prisoner[]> => {
+    const filterPrisonersReleasedWithinNoticePeriod = (prisoners: Prisoner[]): Prisoner[] => {
+      const initialPromptPeriod = filterPrisonersForReleaseWithWeekRange(prisoners, 13, 14)
+      const regularPromptPeriod = filterPrisonersForReleaseWithWeekRange(prisoners, 0, 5)
+      return initialPromptPeriod.concat(regularPromptPeriod)
+    }
+
+    return prisonerService
+      .searchPrisonersByPrison(prisonCode)
+      .then(prisoners => filterPrisonersReleasedWithinNoticePeriod(prisoners))
+      .then(prisoners => caseloadService.mapOffendersToLicences(prisoners))
+      .then(prisoners =>
+        prisoners
+          .filter(
+            prisoner =>
+              prisoner.licenceStatus === LicenceStatus.NOT_STARTED ||
+              prisoner.licenceStatus === LicenceStatus.IN_PROGRESS
+          )
```

Comment:
Are these status values right? Filter to include licences IN_PROGRESS? I would expect only for NOT_STARTED licences to find their way into the email notifications?